readme.md

```diff
@@ -197,6 +197,17 @@ XO is based on ESLint. This project started out as just a shareable ESLint confi
 - [Sublime Text](https://github.com/sindresorhus/SublimeLinter-contrib-xo)
 - [Atom](https://github.com/sindresorhus/atom-linter-xo)
 
+### Vim
+
+You can use [Syntastic](https://github.com/scrooloose/syntastic)'s ESLint checker with the following settings in your `.vimrc` file:
+
+```vim
+let g:syntastic_javascript_eslint_generic = 1
+let g:syntastic_javascript_eslint_exec = 'xo'
+let g:syntastic_javascript_eslint_args = '--compact'
+let g:syntastic_javascript_checkers = ['eslint']
+```
```

Comment:
@rauchg Pretty sure it does, though having both `eslint` and `xo` wouldn't make sense either since they're both going to try to lint javascript files and give you weird results anyway. ViM isn't going to be smart enough to figure out which scripts you want linted as XO and which you want linted as generic ESLint.